emeis/core/urls.py

```diff
@@ -12,7 +12,13 @@
 r.register(r"acls", views.ACLViewSet)
 
 urlpatterns = [
-    url(r"^me", views.MeViewSet.as_view({"get": "retrieve"}), name="me-detail")
+    url(r"^me", views.MeViewSet.as_view({"get": "retrieve"}), name="me-detail"),
+    url(
+        r"^myacls/(?P<pk>[^/.]+)$",
```

Comment:
More of a general input, but we should consider moving from `url` to `path` for new implementations.
https://docs.djangoproject.com/en/2.2/ref/urls/#path

It's nicer and most of all `url` docs state that "It’s likely to be deprecated in a future release."

Reply:
:+1:

Reply:
Absolutely! Done.